2023-risks/m4-inadequate-privacy-controls.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: col-sidebar
+title: "M4: Inadequate Privacy Controls"
+---
+
+# Threat Agents
+
+**Application Specific**
+
+Privacy controls are concerned with protecting Personally Identifiable Information (PII). This information is valuable to attackers for various reasons, e.g., for
+
+- impersonating the victims in cases of fraud,
+- misusing the victims' payment data, or
+- blackmailing the victims with sensitive information.
+
+# Attack Vectors	
+
+**Exploitability MEDIUM**
+
+Typical sources for PII are either protected or hard to access, e.g., the sandbox of the app, the network communication with the server, the app's logs and backups. Some are easier to observe, like URL query parameters and clipboard content.
+
+Obtaining PII thus requires the attacker to breach security on another level. He could eavesdrop on the network communication, access file system, clipboard or logs with a trojan or get his hands on the mobile device and create a backup to analyse.
+
+# Security Weakness	
+
+**Prevalence COMMON** <br/>
+**Detectability EASY**
+
+Risks of privacy violations increase due to careless handling of PII by developers. PII should always be processed with the posibility in mind that an attacker could access communication and storage media. Hence, PII should not be stored or transfered
+unless absolutely necessary. If it is to store or transfer, one should consider encrypting or anonymizing it if feasible. Leakage of PII to uncontrollable sinks, e.g., logs, clipboard, url-parameters, must be prevented.
+
+
+# Technical Impacts	
+
+**Impact LOW**
+
+Privacy violations usually have little technical impact on the system as a whole. Only if the PII includes information like authentication data, it can affect traceability. If user data is manipulated it might render the system unusable for that user.
+
+
+# Business Impacts
+	
+**Impact SEVERE** 
+
+The business impact of privacy violations will typically result in the following at a minimum:
```

Comment:
+1

Regulations are the biggest issue when talking about privacy controls. GDPR, CCPA (California, US), PDPA (Singapore), PIPEDA (Canada), LGPD (Brazil), Data Protection Act 2018 (UK), POPIA (South Africa), PDPL (China) are examples of relevant regulations with known sanctions against companies for not protecting their users' data.